src/controller/python/ChipDeviceController-StorageDelegate.h

```diff
@@ -29,16 +32,19 @@ typedef void (*DeleteKeyValueFunct)(const uint8_t * key);
 namespace chip {
 namespace Controller {
 
-// TODO: Implement this.
 class PythonPersistentStorageDelegate : public PersistentStorageDelegate
 {
 public:
     PythonPersistentStorageDelegate() {}
-    void SetDelegate(PersistentStorageResultDelegate * delegate) override {}
-    void GetKeyValue(const char * key) override {}
-    CHIP_ERROR GetKeyValue(const char * key, char * value, uint16_t & size) override { return CHIP_ERROR_NOT_IMPLEMENTED; }
-    void SetKeyValue(const char * key, const char * value) override {}
-    void DeleteKeyValue(const char * key) override {}
+    void SetDelegate(PersistentStorageResultDelegate * delegate) override;
+    void GetKeyValue(const char * key) override;
+    CHIP_ERROR GetKeyValue(const char * key, char * value, uint16_t & size) override;
+    void SetKeyValue(const char * key, const char * value) override;
+    void DeleteKeyValue(const char * key) override;
```

Comment:
Wait. Why do we introduce some changes to `PersistenStorageDelegate` in this PR ? Seems unrelated to the core of this PR to me. Mind doing that in a separate PR ?

Reply:
Oops, I added them for trying to make CommandSender init/reset aware of DevicePairing state but forgot to remove when given up doing so. Removed

Reply:
We need this to make device api functional for python.